suro-kafka-producer/src/test/java/com/netflix/suro/sink/kafka/TestKafkaSink.java

```diff
@@ -95,6 +99,49 @@ public Object findInjectableValue(Object valueId, DeserializationContext ctxt, B
         });
     }
 
+    /**
+     * open will success because localhost is a resolvable hostname
+     */
+    @Test
+    public void testOpenWithDownBroker() throws Exception {
```

Comment:
Test method name is 'testOpenWithDownBroker' but how are you shutting down kafka broker? You may have to call kafkaServer.after() to shutdown explicitly.